client/packages/common/src/hooks/useDialog/useDialog.tsx

```diff
@@ -160,7 +160,7 @@ export const useDialog = (dialogProps?: DialogProps): DialogState => {
     enableAutocomplete,
     sx = {},
     deleteButton,
-    fullscreen = false,
+    fullscreen,
```

Comment:
took out the default, because this renders the nullish coalescing operator in the following line (239) pointless

```
        fullScreen={fullscreen ?? defaultFullscreen}
```

if the intention is to be defaulting to `false` then the `defaultFullscreen` variable can be removed instead